src/Build/Dockerfiles/Jupyter.Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Argument that takes Algorithm-Image-Name as input
+ARG BASE_IMAGE
+
+# From algorithm image
+FROM $BASE_IMAGE
+
+ARG FOLDER_NAME
+
+# Set the working directory within the container
+WORKDIR /bilayers
+
+# Install the dependencies for the gradio app
+RUN python -m pip install pyyaml jupyter huggingface-hub==0.23.4 pydantic==2.7.4
+
+# Install numpy and opencv-python
+RUN python -m pip install numpy==1.23.0 opencv-python-headless==4.5.3.56 matplotlib==3.5.1
+
+# Add app.py file to the container
+ADD parse/generated_folders/$FOLDER_NAME/generated_notebook.ipynb /bilayers/
+
+# Add __init__.py file for importing the files inside docker-container
+ADD __init__.py /bilayers/
+
+# Export the port
+EXPOSE 8888
```

Comment:
8181? Because 81 is "bi" in making-your-calculator-write-words-in-numbers language, and then have it twice, so it's a bi-instance of bi?

(I can see myself out now)

Reply:
(I could also live with 7878, which would upside down look like BLBL)

Reply:
<img src="https://i.giphy.com/XgXerDpg6U6WSynLc5.webp" width="480" height="355"></img>

Reply:
Whether that gif is in refernce to "BI BI" or "I'll see myself out", I'll leave up to you to decide.